editor/editor_node.cpp

```diff
@@ -7687,7 +7055,8 @@ EditorNode::EditorNode() {
 	default_layout->set_value(docks_section, "dock_4", "FileSystem");
 	default_layout->set_value(docks_section, "dock_5", "Inspector,Node,History");
 
-	for (int i = 0; i < vsplits.size(); i++) {
+	// There are 4 vsplits and 4 hsplits.
+	for (int i = 0; i < 4; i++) {
```

Comment:
I assume you did it because this is now private in another class? I would add a getter. In case we change something about our splits it would be a pain to track down hardcoded values.

Reply:
The vsplits and hsplits are made in this function, so I figured it would be fine with just a comment.
Also there would be no use for a matching hsplit count getter, since those values are hardcoded anyway.

I'll add a `get_vsplit_count()` method anyway just so it's easier to track down.